amacc.c

```diff
@@ -441,6 +441,16 @@ void next()
 
 char fatal(char *msg) { printf("%d: %s\n", line, msg); exit(-1); }
 
+int popcount32b(int n)
```

Comment:
Variable `n` is global, and you CAN NOT use it for function prototype.

Instead, you should rewrite as following:
```c
int popcount32b(int x)                                                                                                                                        
{
    x -= x >> 1 & 0x55555555;
    x = (x >> 2 & 0x33333333) + (x & 0x33333333);
    x = (x >> 4) + (x & 0x0f0f0f0f);
    x += x>> 8;
    x += x >> 16;
    return x & 0x1f;
}
```

Reply:
With the above modification, AMaCC is able to bootstrap itself.